code/src/Components/QuestionRadio.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import React from 'react';
+
+const QuestionRadio = (props) => {
```

Comment:
Maybe there is a reason for doing it the way you do, but I think you can just add your props like this:
const QuestionRadio = ({ radio, setRadio } ) => {